src/polkadot.scss

```diff
@@ -171,23 +171,6 @@ $navbar-dark-hover-color: $gray-900;
 // Radii
 $border-radius: 0.5rem;
 
-// account dropdown
-.account-dropdown-toggle.dropdown-toggle::after {
-  display: none; // hide default bootstrap dropdown caret
-}
-.account-toggle-item {
-  background-color: #fff !important;
-}
-.account-toggle-item:hover {
-  background-color: #f9fafb !important;
-  box-shadow: 0 0.25rem 0.5rem rgba(#000, 0.1);
-}
-// Balance Text
-.balance-text {
-  font-size: 0.85rem !important;
-  color: #757575 !important;
-}
-
 .image-box {
```

Comment:
need to add this to fix the ledger account import page in generate gift

Reply:
![Uploading Screen Shot 2021-06-22 at 8.41.07 AM.png…]()

Reply:
![Screen Shot 2021-06-22 at 8 41 07 AM](https://user-images.githubusercontent.com/5099795/122956377-ccf0e980-d335-11eb-8000-ae7424205de7.png)